src/python/pants/option/options.py

```diff
@@ -317,14 +317,25 @@ def for_scope(self, scope, inherit_from_enclosing_scope=True):
 
     return values
 
-  def get_fingerprintable_for_scope(self, scope):
+  def get_fingerprintable_for_scope(self, scope, include_passthru=False):
```

Comment:
This is a public API. Used in 1 spot inside pants, and I've adjusted that call site, but who knows how it's used elsewhere. I agree, probably not used elsewhere, but `:API: public` is a commitment to pretend it is.